### PR TITLE
bug fix for recording.

### PR DESCRIPTION
**Pull request by ziquanyu**
bug fix for recording crush.

### CODE REVIEW

**Comment by ziquanyu** [comment]
@emilwestergren Can you review this when you have time. Thanks!

File: vidhance/sensors/SensorReader.cpp [hunk 1]
```diff
@@ -13,14 +13,23 @@
 
 void SensorReader::printSensors() {
 	ASensor const* const* list;
-	ssize_t count = ASensorManager_getSensorList(ASensorManager_getInstance(), &list);
+	ssize_t count;
+#if ANDROID_VERSION_MAJOR < 6
```
Comment (emilwestergren) [inline vidhance/sensors/SensorReader.cpp]: Create a static function `getSensorManager` instead and you only need to do this check once.

Comment (ziquanyu) [inline vidhance/sensors/SensorReader.cpp]: OKay

**Comment by ziquanyu** [comment]
@emilwestergren it's done, I added one adapted file in a separate commit.

**Pull request merged**